packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -148,6 +148,12 @@ class _MenuAnchorScope extends InheritedWidget {
 ///
 /// ** See code in examples/api/lib/material/menu_anchor/menu_anchor.3.dart **
 /// {@end-tool}
+///
+/// Visual density should not affect menu vertical padding according to
+/// Material Design specs.
+/// Per the Material Design team: don't allow the VisualDensity adjustment to
+/// reduce the width of the left/right padding. If we did, VisualDensity.compact,
+/// the default for desktop/web, would reduce the horizontal padding to zero.
```

Comment:
I think this information is a bit too detailed for the `MenuAnchor` class here. Also the format is not in the style of public documentation. I recommend moving it to `style` and rephrase it such as:
```dart
  ///
  /// The [MenuStyle.visualDensity] setting only affects horizontal padding, and it will never make it negative. Vertical padding is not affected at all.
```